recipes/docling/recipe.yaml

```diff
@@ -0,0 +1,80 @@
+context:
+  name: docling
+  version: 2.6.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/docling-${{ version }}.tar.gz
+  sha256: 282755472a1a1d4a4088d794c6a89f399a5b7fbcea8b5f45012c441670b538d5
+  patches:
+    # lower the scipy lower bounds to match with c-f and avoid conflicts
+    - fix-deps.patch
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - docling = docling.cli.main:app
+
+requirements:
+  host:
+    - python ${{ python_min }}
+    - poetry-core
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - pydantic >=2.0.0,<3.0.0
+    - docling-core >=2.3.0,<3.0.0
+    - docling-ibm-models >=2.0.3,<3.0.0
+    - deepsearch-glm >=0.26.1,<0.27.0
+    - filetype >=1.2.0,<2.0.0
+    - pypdfium2 >=4.30.0,<5.0.0
+    - pydantic-settings >=2.3.0,<3.0.0
+    - huggingface_hub >=0.23,<1
+    - requests >=2.32.3,<3.0.0
+    - easyocr >=1.7.0,<2.0.0
+    - docling-parse >=2.0.2,<3.0.0
+    - certifi >=2024.7.4
+    - rtree >=1.3.0,<2.0.0
+    - scipy >=1.13.1,<2.0.0
+    - pyarrow >=16.1.0,<17.0.0
+    - typer >=0.12.5,<0.13.0
+    - python-docx >=1.1.2,<2.0.0
+    - python-pptx >=1.0.2,<2.0.0
+    - beautifulsoup4 >=4.12.3,<5.0.0
+    - pandas >=2.1.4,<3.0.0
+    - marko >=2.1.2,<3.0.0
+    - openpyxl >=3.1.5,<4.0.0
+  run_constraints:
+    - tesserocr >=2.7.1,<3.0.0
+
+tests:
+  - python:
+      imports:
+        - docling
+      # `pip check` fails because:
+      # - "easyocr 1.7.1 requires ninja, which is not installed". See https://github.com/conda-forge/easyocr-feedstock/pull/12#issuecomment-2484602055
+      # - "scikit-image 0.24.0 has requirement numpy>=1.23, but you have numpy 1.22.4."
+      pip_check: false
```

Comment:
> I had to disable `pip check` cuz the failures are due to downstream deps (nothing I can patch). Let me know what you think.

I'm assuming the comment here is the entirety of the `pip check`. You repushed quickly enough that the CI for https://github.com/conda-forge/staged-recipes/pull/28270/commits/3afad17c4a3551349ca27872ea8ffc6ef815bf02 got cancelled and thus deleted.

I see you're already aware of https://github.com/conda-forge/easyocr-feedstock/pull/12. For the scikit-image thing, that's a consequence of pulling in pre numpy 2.0 dependencies (it's an inaccuracy on the scikit-image feedstock that has already been [fixed](https://github.com/conda-forge/scikit-image-feedstock/commit/bbfa44b7e202bf0de5b6d1f15a58bff99cb39eec)). In this case it's not worth to create a migration branch to fix this on the scikit-image feedstock, but this should be re-enabled as soon as the numpy 2.0 migrator comes by.

Reply:
Yes that's it.

Thanks for the details on skimage/numpy.